packages/create-sitecore-jss/src/templates/vue/scripts/install-pre-push-hook.js

```diff
@@ -1,28 +1,32 @@
-﻿const fs = require('fs');
+﻿﻿const fs = require('fs');
+const path = require('path');
+const { promisify } = require('util');
+const { exec } = require('child_process');
+const chalk = require('chalk');
```

Comment:
I think vue is the only one without chalk. Would need to add it to devDependencies.